query/src/main/java/org/infinispan/query/impl/massindex/MassIndexerProgressMonitor.java

```diff
@@ -23,7 +23,7 @@ public class MassIndexerProgressMonitor {
     * Logs progress of indexing job every 50 documents written.
     */
    public MassIndexerProgressMonitor(TimeService timeService) {
-      this(50, timeService);
+      this(200, timeService);
```

Comment:
As @wfink said I would increase this even more (1000?). Also adjust the comment above